src/lib/modernExtend.ts

```diff
@@ -1150,9 +1150,10 @@ export interface ElectricityMeterArgs {
     power?: false | MultiplierDivisor,
     voltage?: false | MultiplierDivisor,
     energy?: false | MultiplierDivisor
+    reporting?: boolean
```

Comment:
```suggestion
    configureReporting?: boolean
```

To stay consistent with other modern extends

Reply:
Done!